avalon/style/style.qss

```diff
@@ -505,14 +510,6 @@ QPlainTextEdit
     border: 1px solid #3A3939;
 }
 
-QHeaderView::section
-{
-    background-color: #3A3939;
-    color: silver;
-    padding-left: 4px;
-    border: 1px solid #6c6c6c;
-}
```

Comment:
This section was duplicated, there are one exact definition with more attributes down below this `style.qss` file.